dbt-cta/macros/cross_db_utils/datatypes.sql

```diff
@@ -75,9 +75,10 @@
 
 
 {# int  ------------------------------------------------- #}
-{% macro default__type_int() %}
+{# BQ does not have a "signed" data type, so this is commented out #}
+{# {% macro default__type_int() %}
     signed
```

Comment:
out of curiosity, was this causing a problem while it was not commented-out?

Reply:
It wasn't invoked anywhere in our project, but somewhere in an internal elementary thing, they called `CAST as default__type_int()`. This template rendered as `CAST as SIGNED` which failed. With this commented out, it renders as `CAST as INTEGER`, which is correct.

Reply:
oh that's sneaky!! thank you for the lore 🐸